### ISSUE TITLE
Use of Milestones

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeREGNonProductIssues#22]
It is useful for reporters to assign a tentative milestone to a new story. This provides us all with a sense of the importance of an story in the mind of the originator. We are currently working on milestone 1.4. anything with that milestone has to be completed BEFORE we can move to 1.5. We have future milestones 1.5, 1.6, 1.7 and 2. Milestones 1.1, 1.2 and 1.3 have been completed and should not be used for new stories
There are many labels that can be assigned,these are useful in reviewing and assigning work or developing the story further; for urgent issues within a milestone the Urgent label should be used.

**Comment by PatReynolds** [comment]
Agreed.  Can anything be done to prompt/enforce this?

**Comment by Captainkirkdawson** [comment]
Enforcement not practical; just good practice

**Comment by PatReynolds** [comment]
I would like a way of grouping stories within a milestone (for example, GAP,  Open Data, Images, and Communications within 1.5 - as a way of helping to sort priorities.  Communications already has a tag, could we have more?

**Comment by Captainkirkdawson** [comment]
Pat
You can add new labels.
Go to the repository on github. In the case you are interested in is FreeregProductIssues
![pat1](https://cloud.githubusercontent.com/assets/2874694/16269752/cd0ea0a4-3850-11e6-8e3a-bb04af13b716.JPG)
Select Issues
![pat2](https://cloud.githubusercontent.com/assets/2874694/16269764/e552e670-3850-11e6-8791-7059440e27b1.JPG)
The listing of issues comes up but at the top select the labels button
![pat3](https://cloud.githubusercontent.com/assets/2874694/16269784/09d35e76-3851-11e6-8e7b-661fe59911c9.JPG)

A listing of all available labels comes up. You want a new one. Click the New Label button and add what you want
